.npmrc

```diff
@@ -0,0 +1,3 @@
+engine-strict=true
```

Comment:
This is critical bit, otherwise olde node may still be used and we endup debugging unexpected behaviors. Yep, without it node would only raise a warning when `>=` condition is not met.